src/SDKS/erc20/index.ts

```diff
@@ -140,6 +144,26 @@ export namespace CryptoWallet.SDKS.ERC20 {
       });
     }
 
+    estimateGas(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class method 'estimateGas' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/app/atlascity/cryptowallet-js/pullRequest?prid=3649440)